src/doc/trpl/installing-rust.md

```diff
@@ -1,24 +1,24 @@
 % Installing Rust
 
 The first step to using Rust is to install it! There are a number of ways to
-install Rust, but the easiest is to use the `rustup` script. If you're on Linux
-or a Mac, all you need to do is this (note that you don't need to type in the
-`$`s, they just indicate the start of each command):
+install Rust, but first we ought to go over the officially supported architecture's.
```

Comment:
I'm not totally sold on putting this first, but I'm also not against it exactly. Hm.